build/format-notecards.js

```diff
@@ -0,0 +1,22 @@
+function formatNotecards($) {
+  $(".note.notecard").each(() => {
+    const $h4 = $("h4");
+    const id = $h4.attr("id");
+    const $p = $("p");
+    const $replacer = $(`<strong>${$h4.text()}:</strong>`);
+
+    if ($h4.length) {
+      $h4.empty();
+      $h4.replaceWith($replacer);
+      $p.prepend($replacer);
+      // if the h4 has id="note" add it as <strong id="note">
+      if (id === "note") {
+        $replacer.attr("id", id);
+      }
+    }
+
+    return;
+  });
+}
```

Comment:
I don't think this is correct. That `$h4` isn't the `<h4>` inside the `div.note.notecard`. Something like this is more correct:

```js
$('div.notecard h4').each((i, element) => {
  const h4 = $(element)
  const text = h4.text()
  const p = $('p:first', h4.parents('div.notecard'))
  p.html(`<strong>${text}</stong> ${p.html()}`)
  h4.empty()
})
```

Reply:
The above logic works to essentially prepend the `<strong>` tag at the beginning of the first `<p>` tag. But the emptied `<h4>` is still hanging around. That's why I opted for the `replaceWith` usage as well.

<img width="1255" alt="Screen Shot 2021-08-04 at 8 57 24 AM" src="https://user-images.githubusercontent.com/48612525/128214116-74f4ecf2-dafa-46c9-bb21-d7d2c57a408b.png">

Reply:
Ya just adding `h4.remove()` after we empty it does the trick. Using `replaceWith` was kind of hacky anyways.

```js
$(".note.notecard h4").each((_, element) => {
    const h4 = $(element);
    const text = h4.text();
    const p = $("p:first", h4.parents("div.notecard"));
    p.html(`<strong>${text}:</strong> ${p.html()}`);
    h4.empty();
    h4.remove();
  });
```

<img width="1262" alt="Screen Shot 2021-08-04 at 9 02 11 AM" src="https://user-images.githubusercontent.com/48612525/128214834-e44ec482-972f-44b6-9567-e68b03d88699.png">